packages/yoroi-extension/features/step_definitions/common-steps.js

```diff
@@ -245,13 +276,24 @@ Given(/^There is a Byron wallet stored named ([^"]*)$/, async function (walletNa
 Given(/^I have completed the basic setup$/, async function () {
   // language select page
   await this.waitForElement('.LanguageSelectionForm_component');
-  await this.click('.LanguageSelectionForm_submitButton');
+  await this.click('//button[text()="Continue"]', By.xpath);
+ // await this.click('.LanguageSelectionForm_submitButton');
 
   // ToS page
   await this.waitForElement('.TermsOfUseForm_component');
-  await this.click('.SimpleCheckbox_check');
-  await this.click('.TermsOfUseForm_submitButton');
+  //await sleepTh(2000);
+
+  const TOS_class_element = await this.driver.findElement(By.css('.TermsOfUseForm_component'));
+
+  const checkbox = await TOS_class_element.findElement(By.xpath('//input[@type="checkbox"]'));
 
+  await checkbox.click();
+  
+  
+ // this.click('//input[@type="checkbox"]', By.xpath);
+  //await this.click('.SimpleCheckbox_check');
```

Comment:
if we will not use this code, it is fine to remove it completely :)

Reply:
Done